pepper-apis/src/main/java/org/broadinstitute/ddp/export/json/structured/ParticipantProfile.java

```diff
@@ -24,6 +24,8 @@
     private boolean doNotContact;
     @SerializedName("createdAt")
     private long createdAt;
+    @SerializedName("shouldSkipLanguageProfile")
+    private boolean shouldSkipLanguageProfile;
```

Comment:
Updating these data export objects means we're changing the structure of the Elasticsearch documents, which means we'll need to refresh Elasticsearch indices when we roll out this change. I'll recommend holding off on doing that unless we have a requirement to show this particular setting in DSM. Even if there is a need, we should coordinate with DSM to make sure it shows up there.